src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big">
+      <label class="wrapper">
+        <img 
+          src="./images/Search.svg" 
+          alt="search" 
+          class="img-big"
+        >
+        <input 
+          placeholder="Try &quot;Los Angeles&quot"
+          class="big-search"
+          data-qa="keypress"
+        >
+      </label>
+    </form>
```

Comment:
let's make classes more meaningful. for example
```suggestion
    <form class="search-form" data-qa="big">
      <label class="form-field">
        <img 
          src="./images/Search.svg" 
          alt="search" 
          class="search-icon"
        >
        <input 
          placeholder="Try &quot;Los Angeles&quot"
          class="form-input"
          data-qa="keypress"
        >
      </label>
    </form>
```
also pay attention on this point from  [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
![Screenshot from 2023-04-01 15-10-21](https://user-images.githubusercontent.com/33029864/229288272-c25a26ec-ccb1-4a97-8a48-5467fb5d919f.png)

Reply:
+